### PR TITLE
Fix password resets

### PR DESCRIPTION
**Pull request by igorbabko**

### CODE REVIEW

**Comment by jasonvarga** [comment]
What is this fixing?

File: config/users.php [hunk 1]
```diff
@@ -74,7 +74,7 @@
     */
 
     'passwords' => [
-        'resets' => config('auth.defaults.passwords'),
+        'users' => config('auth.defaults.passwords'),
```
Comment (damcclean) [inline config/users.php:77]: @jasonvarga Would it be considered a breaking change now that Statamic 3 is out if we changed the key of a configuration option?

**Comment by damcclean** [comment]
I think this pull request is related to #2133, could you confirm @igorbabko?

**Comment by igorbabko** [comment]
> What is this fixing?

@jasonvarga The password reset form fails to submit - https://d.pr/v/6hpTaW when storing users in the database.

This PR fixes it.

**Comment by igorbabko** [comment]
> I think this pull request is related to #2133, could you confirm @igorbabko?

@damcclean Not really, this fixes the following excepting when trying to submit password reset form (when storing users in db):

Screen Shot 2020-09-15 at 14 24 33

**Comment by jasonvarga** [comment]
(Hey cool wayback machine extension thingy)

**Comment by edalzell** [comment]
I can confirm this fixes it for NON-db users as well @jasonvarga

**Comment by jasonvarga** [comment]
Thank you for the PR, but it required a different solution. Should be fixed now.

**Comment by igorbabko** [comment]
> (Hey cool wayback machine extension thingy)

@jasonvarga Yeah) I believe it's a default feature of Brave Browser - https://brave.com/

**Pull request closed without merging**